crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/instructions_v2/instructions_v2.rs

```diff
@@ -280,15 +287,16 @@ mod tests {
     fn extract_error_from_addr_uses_invalid_instructions_string_if_same_network(
     ) {
         assert_eq!(
-            CommonError::from_address_error("account_rdx128y6j78mt0aqv6372evz28hrxp8mn06ccddkr7xppc88hyvynvjdwr".to_owned(), NetworkID::Mainnet),
-            CommonError::InvalidInstructionsString { underlying: "Failed to determine why an address was invalid".to_owned() }
+            CommonError::from_address_error("account_rdx128y6j78mt0aqv6372evz28hrxp8mn06ccddkr7xppc88hyvynvjdwr".to_owned(), NetworkID::Mainnet, "fallback".to_owned()),
+            CommonError::InvalidInstructionsString { underlying: "fallback".to_owned() }
         );
     }
 
     #[test]
     fn extract_error_from_error_non_gen_err() {
         assert_eq!(
             CommonError::from_scrypto_compile_error(
+                "invalid_manifest",
```

Comment:
"this is a super duper invalid manifest" is more clear a test string IMO.

Reply:
because at first I thought this was suppose to match some expected enum value or so